Vagrantfile

```diff
@@ -16,6 +16,9 @@ end
 
 # Default vagrant config.
 vconfig = {
+  'beet_home' => '/beetbox',
+  'beet_root' => '/var/beetbox',
```

Comment:
I think having abother directory called `beetbox` is just confusing. Can this be called `app`?

Also, why is it in `/var`?

Feels much counterintuitive locations and naming. It think `/www/app` is quite generic and in the expected location.